test/es6/ES6Promise.js

```diff
@@ -82,6 +82,13 @@ var tests = [
             assert.areEqual('function', typeof descriptor.value, "typeof Promise.prototype.then === 'function'");
             assert.areEqual(2, Promise.prototype.then.length, "Promise.prototype.then.length === 2");
 
+            var descriptor = Object.getOwnPropertyDescriptor(Promise.prototype, 'finally');
+            assert.isTrue(descriptor.writable, "Promise.prototype.finally.writable === true");
+            assert.isFalse(descriptor.enumerable, "Promise.prototype.finally.enumerable === false");
+            assert.isTrue(descriptor.configurable, "Promise.prototype.finally.configurable === true");
+            assert.areEqual('function', typeof descriptor.value, "typeof Promise.prototype.finally === 'function'");
+            assert.areEqual(1, Promise.prototype.finally.length, "Promise.prototype.finally.length === 1");

```

Comment:
can you also add an assert that the `.name` is `"finally"`?

Reply:
Done - and added equivalents for .then and .catch as they didn't exist either.